packages/salesforcedx-apex-debugger/package.json

```diff
@@ -47,8 +51,8 @@
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "test": "npm run test:unit && npm run test:integration",
-    "test:unit": "node ./node_modules/nyc/bin/nyc.js ./node_modules/mocha/bin/_mocha --recursive out/test/unit --exit --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaUnitTestsConfig.json",
-    "test:integration": "node ./node_modules/cross-env/dist/bin/cross-env.js VSCODE_NLS_CONFIG={} ./node_modules/nyc/bin/nyc.js ./node_modules/mocha/bin/_mocha --recursive out/test/integration --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaIntegrationTestsConfig.json"
+    "test:unit": "jest --coverage",
+    "test:integration": "jest --coverage -c ./jest.integration.config.js"
```

Comment:
This is the standard jest config for all projects. If a project is using the vscode test running for integration tests that was left in place.

Reply:
jest and --coverage configurations are defined in jest.base.config file, which is not included in this PR. It's a part of PR part-3 4429.